source/include/localisation/cs_TrackableId.h

```diff
@@ -23,15 +24,37 @@
  */
 struct __attribute__((__packed__)) TrackableId {
 	static constexpr uint8_t SIZE = 6;
-	uint8_t bytes[SIZE] = {0};
+
+	/**
+	 * Note: mac bytes are stored in reverse order of how they
+	 * are read on the nrf connect app.
+	 */
+	uint8_t bytes[SIZE] = {0x00, 0x00, 0x00, 0x00, 0x00, 0x00};
```

Comment:
Is trackableId always a mac address? If so, use `MAC_ADDRESS_LEN` from `cs_Packets.h`. Otherwise, don't mention mac addresses here?
If this is just temporary, you can change it later.

Reply:
TrackableId will be refactored when continuing the filter implementation. It will go down to 2 bytes, representing a fingerprint. (Perhaps 3 bytes for a fingerprint-index pair which might be necessary for lookup.)